UI/Pages/SchoolLedTutoring.cshtml

```diff
@@ -1,9 +1,63 @@
+  Layout = "_Layout";
+  ViewData["Title"] = "SchoolLedTutoring - Compare National Tutoring Options";
 }
-
-<h1 class="govuk-heading-xl">School-led tutoring</h1>
-
-<p class="govuk-body">School-led tutoring content</p>
\ No newline at end of file
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <a href="/" data-testid="home-link" class="govuk-back-link">Home</a>
+    <h1 class="govuk-heading-l" data-testid="school-led-header">Employ your own tutor</h1>
+    <p class="govuk-body">You can employ someone directly as a tutor for your school, for example a member of teaching
+      staff, a parent or a retired teacher. They’ll provide high-quality 1 to 1 and small-group sessions (no more than 6
+      pupils).
+    </p>
+    <h2 class="govuk-heading-l">Training for tutors
+    </h2>
+    <p class="govuk-body">
+      Free training in tutoring is:
+    </p>
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-4">
+      <li>compulsory for tutors who don’t have QTS (qualified teacher status) unless they’ve completed the training in a
+        previous year
+      </li>
+      <li>
+        optional for tutors with QTS or who’ve already completed the training
+      </li>
+    </ul>
+    <p class="govuk-body">
+      Training can be tailored to meet the requirements of your pupils.
+    </p>
+    <p class="govuk-body">
+     You’ll be able to access a site where you can book training for tutors from 1 September 2022..
+    </p>
+    <h2 class="govuk-heading-l">What it costs
+    </h2>
+    <p class="govuk-body">
+      The cost of tutoring per pupil per hour depends on the size of tutor group and skills of the tutor. The agreed
+      maximum rates for which you’ll get the 60% subsidy are:
+    </p>
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-4">
+      <li>£18 per hour in a mainstream school - you’ll pay £7.20
+      </li>
+      <li>
+        £47 per hour in a non-mainstream school - you’ll pay £18.80
+      </li>
+    </ul>
+    <p class="govuk-body">
+      For more details and examples <a href="/funding-and-reporting" data-testid="funding-allocation-link" class="govuk-link">read the information on funding.</a>
+    </p>
+    <h2 class="govuk-heading-l">Checks you need to do
+    </h2>
+    <p class="govuk-body">
+      You must carry out the following checks if you employ someone as a tutor who is not a current member of school
+      staff:
+    </p>
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-4">
+      <li>references</li>
+      <li>
+        <a href="https://www.gov.uk/find-out-dbs-check/y/caring-for-or-working-with-children-under-18-or-working-in-a-school/teaching-or-caring-for-children/working-in-a-school-nursery-children-s-centre-or-home-detention-service-young-offender-institution-or-childcare-premises/yes"
+          data-testid="dbs-check-link" class="govuk-link">DBS enhanced check with barred lists</a>
```

Comment:
As this is a an address outside of our url origin I think it should be in a new tab